cambios interfaz dspace

jaimescript opened the pull request:
cambios interfaz dspace

Commit:
cambios interfaz dspace

cambios interfaz dspace

colivaresp committed:
Delete README.md

colivaresp commented on README.md:
```diff
@@ -0,0 +1,3 @@
+# dspace_concytec_customized
+
```
Eliminar este archivo añadido. No es necesario añadirlo.

colivaresp commented on build.properties:
```diff
@@ -22,13 +22,13 @@
 # to install DSpace. NOTE: this value will be copied over to the
 # "dspace.dir" setting in the final "dspace.cfg" file. It can be
 # modified later on in your "dspace.cfg", if needed.
-dspace.install.dir=/dspace
```
No se debió enviar cambios a este archivo en el pull requests. Regresarlos a su valor anterior antes de enviar nuevo pull request.

colivaresp commented on dspace-cris/jspui-webapp/src/main/webapp/dspace-cris/commonDetailsPage.jsp:
```diff
@@ -30,7 +30,6 @@
 	Map<String, IComponent> mapInfo = ((Map<String, IComponent>)(request.getAttribute("components"))); 
 	boolean showBadgeCount = ConfigurationManager.getBooleanProperty("cris", "webui.tab.show.count.for.firstcomponent", false);
 %>
-	
```
No se debe hacer cambios en este archivo

colivaresp commented on dspace-cris/jspui-webapp/src/main/webapp/dspace-cris/stats/collection/collectionReport.jsp:
```diff
@@ -44,8 +44,7 @@
 	</script>
 </c:set>
-
-
```
No se debe tocar innecesariamente este archivo. Regresarlo a su contenido anterior.

colivaresp commented on dspace-cris/jspui-webapp/src/main/webapp/dspace-cris/stats/common/changeRange.jsp:
```diff
@@ -7,6 +7,7 @@
     https://github.com/CILEA/dspace-cris/wiki/License
 
 --%>
+
```
No se debe tocar innecesariamente este archivo. Regresarlo a su contenido anterior.

colivaresp commented on dspace-cris/jspui-webapp/src/main/webapp/dspace-cris/stats/common/upload.jsp:
```diff
@@ -21,7 +21,7 @@
 <div id="statstabs"> 
 <div id="statstab-content">
 	<div id="statstab-content-time" class="statstab-content-item">		
-		<%@include file="time.jsp"%> 
```
No se debe tocar innecesariamente este archivo. Regresarlo a su contenido anterior.

colivaresp commented on dspace-jspui/src/main/webapp/browse/full.jsp:
```diff
@@ -29,6 +29,14 @@
 <%@ page import="org.dspace.app.webui.util.UIUtil" %>
 <%@ page import="org.dspace.eperson.EPerson" %>
 
+<%@ page import="javax.servlet.jsp.jstl.fmt.LocaleSupport" %>
```
Este y los demás jsp de la carpeta dspace-jspui/src/main/webapp/ deben ser copiados a /dspace/modules/jspui/src/main/webapp/ donde debe quedar la versión personalizada. La versión original debe ser restaurada en dspace-jspui/src/main/webapp/ 

P.ej.:
/dspace-jspui/src/main/webapp/browse/full.jsp  --> /dspace/modules/jspui/src/main/webapp/browse/full.jsp

colivaresp commented on dspace-jspui/src/main/webapp/layout/header-default.jsp:
```diff
@@ -79,7 +79,10 @@
 		<link rel="stylesheet" href="<%= request.getContextPath() %>/static/css/bootstrap/dspace-theme.css" type="text/css" />
 		<link rel="stylesheet" type="text/css" href="<%= request.getContextPath() %>/css/bootstrap-datetimepicker.min.css" />
-			<link rel="stylesheet" type="text/css" href="<%= request.getContextPath() %>/static/css/select.dataTables.min.css" />
+			<link rel="stylesheet" type="text/css" href="<%= request.getContextPath() %>/static/css/select.dataTables.min.css" />			<link rel="stylesheet" type="text/css" href="<%= request.getContextPath() %>/static/css/select.dataTables.min.css" />
```
Revisar esto, al parecer se ha duplicado la línea en la misma línea.

colivaresp reviewed (changes requested):
Por favor corregir las observaciones indicadas.

Commit:
cambios simples de .jsp

correcciones observadas sobre:

README
build.properties
commonDetailsPage.jsp en carpeta dapce-cris
collectionReport.jsp en carpeta dapce-cris
changeRange.jsp en carpeta dapce-cris
upload.jsp en carpeta dapce-cris

Commit:
Merge branch 'dev-CONCYTEC-repositorio' of https://github.com/pacinete/DSpace into dev-CONCYTEC-repositorio

Commit:
move jsp pages to modules

move jsp pages to modules
